manifests/l/Lupinho/HardlinkBackup/2.2.23/Lupinho.HardlinkBackup.installer.yaml

```
@@ -0,0 +1,35 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Lupinho.HardlinkBackup
+PackageVersion: 2.2.23
+InstallerLocale: en-US
+InstallerType: exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /quiet /qn
+  SilentWithProgress: /passive /qn
+  Log: /log <LOGPATH>
+  Upgrade: /update
+Installers:
+- Architecture: x86
+  InstallerUrl: https://www.lupinho.net/software/HardlinkBackup/Current/HardlinkBackupSetup_x86.exe
+  InstallerSha256: DBA261D879AC52AEF4A2F8E6B96E5E6F92918C43EB9A7C298DA81F0841E66D37
+  AppsAndFeaturesEntries:
+  - DisplayName: HardlinkBackup
+    Publisher: Lupinho.Net
+    DisplayVersion: 2.2.23
+    ProductCode: '{83BA37E7-1169-4E35-8BD6-69E2CFCA8ADD}'
+- Architecture: x64
+  InstallerUrl: https://www.lupinho.net/software/HardlinkBackup/Current/HardlinkBackupSetup_x64.exe
+  InstallerSha256: CAD5EAA09E2828421BC789F8F04D5AD44624AD11A3203230F0C8690B8B39F2C8
+  AppsAndFeaturesEntries:
+  - DisplayName: HardlinkBackup (64 bit)
+    Publisher: Lupinho.Net
+    DisplayVersion: 2.2.23
+    ProductCode: '{83BA37E7-1169-4E35-8BD6-69E2CFCA8ADD}'

```

```suggestion
  - DisplayName: HardlinkBackup (64 bit)
    Publisher: Lupinho.Net
    ProductCode: '{83BA37E7-1169-4E35-8BD6-69E2CFCA8ADD}'
```
DisplayVersion should not be used when it's equal to PackageVersion. See https://github.com/microsoft/winget-pkgs/pull/65816#issue-1301301147